### PR TITLE
Update DG with sort tasks by date command

### PR DESCRIPTION
**Pull request by juayhee**
Related to #127

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -959,6 +959,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
+**Use case: Sort task list by date**
+
+**MSS**
+
+1. User requests to sort the task list by date.
```
Comment (AAlghrairy) [inline docs/DeveloperGuide.md:966]: Consider adding an extension here for when the task list is empty.

Comment (AAlghrairy) [inline docs/DeveloperGuide.md:966]: Consider adding an extension for when the task list is empty.

**Review by AAlghrairy — COMMENTED** [review]
Looks good aside from a couple suggestions listed below.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -959,6 +959,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. Coach2k22 shows the updated task list, sorted with the tasks with the earliest deadline at the top.
+
```
Comment (AAlghrairy) [inline docs/DeveloperGuide.md:968]:
```suggestion
    Use case ends.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -965,7 +965,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1. User requests to sort the task list by date.
 2. Coach2k22 shows the updated task list, sorted with the tasks with the earliest deadline at the top.
-
+    Use case ends.
```
Comment (AAlghrairy) [inline docs/DeveloperGuide.md:968]:
```suggestion

    Use case ends.
```

Comment (AAlghrairy) [reply docs/DeveloperGuide.md:968]: I meant to include have a new line before `Use Case ends.` (make sure to have the format be the same as the previous use cases).

**Comment by simjunheng** [comment]
@juayhee Do work on the comments given by Abdul and let me know once you have done so. I will merge it afterwards

**Comment by juayhee** [comment]
> @juayhee Do work on the comments given by Abdul and let me know once you have done so. I will merge it afterwards

Added content according to Abdul's comments. Should be ready to merge!

**Pull request merged**